docs/monitoring.md

```diff
@@ -229,10 +229,28 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>A list of all applications</td>
   </tr>
   <tr>
+     <td></td>
+     <td><code>?status=[completed|running]</code> list only applications in the chosen state</td>
+  </tr>
+  <tr>
+     <td></td>
+     <td><code>?minDate=[date]</code> earliest date/time to list. Examples:
+     <br><code>?minDate=2015-02-10</code>
```

Comment:
super-nit: should these be `<br/>`?  I guess no browsers care about the `/` anymore, but might as well ...